abci/tests/test_app/main.go

```diff
@@ -63,7 +63,8 @@ func testCounter() {
 		if err := cmd.Wait(); err != nil {
 			log.Printf("error while waiting for cmd to exit: %v", err)
 		}
-
+	}()
+	defer func() {
```

Comment:
I had to undo the combining of these two defers because it was causing CI to not run correctly.

Reply:
Did you remember to reverse the order? Defers run in reverse order (the last one runs first).

Reply:
Thanks for the pointer.